Examples/Io/Root/include/ActsExamples/Io/Root/RootNuclearInteractionParametersWriter.hpp

```diff
@@ -0,0 +1,80 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Framework/WriterT.hpp"
+#include "ActsExamples/Io/Root/detail/NuclearInteractionParametrisation.hpp"
+
+#include <mutex>
+#include <vector>
+
+namespace ActsExamples {
+
+struct ExtractedSimulationProcess;
+
+/// @brief This class takes fractions of recorded events that represent the
```

Comment:
The `@brief` is not needed. The first sentence in a doxygen block will automatically be used for that.

Reply:
removed